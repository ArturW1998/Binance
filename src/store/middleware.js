import logger from 'redux-logger';

const middleware = [];

if (process.env.NODE_ENV === 'development') {
  middleware.push(logger);
}

export default middleware;
