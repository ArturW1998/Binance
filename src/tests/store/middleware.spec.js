import middleware from '../../store/middleware';

describe('middleware', () => {
  it('should return empty middleware without development mode', () => {
    const expectedData = [];

    expect(middleware).toEqual(expectedData);
  });

  it('returns properly', () => {
    expect(middleware).toMatchSnapshot();
  });
});
