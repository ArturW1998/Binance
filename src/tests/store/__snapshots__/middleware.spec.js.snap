// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`middleware returns properly 1`] = `Array []`;
